src/legacy/core_plugins/data/public/index_patterns/fields/field.ts

```diff
@@ -31,6 +31,11 @@ import { IndexPattern } from '../index_patterns';
 
 import { getKbnFieldType } from '../../../../../../plugins/data/public';
 
+interface FieldSubType {
```

Comment:
I wish we had a global types package. It makes me nervous when we have to define types in various places but I've had to do the same.

Reply:
yeah, unfortunately I'm just having to update this everywhere a `Field` type exists